src/LaboratoryResults-3-0/Cert/Kickstart-MO/Send/1-send-nierfunctieNHG-MO.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/testscript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TestScript xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript"
+    nts:scenario="client">
+    
+    <id value="CERT-xis-1-prescription"/>
+    <version value="R4-3.0"/>
+    <name value="LaboratoryResults - LAB-LRS - Scenario 1 - Send a NHG coded laboratory result along with a medication prescription."/>
+    <description value="Scenario 1 - Send a NHG coded laboratory result along with a medication prescription"/>
+    
+    <nts:fixture id="CERT-scenario-1-prescription-fixture"
+        href="resources-to-sendreceive/lr-slr-KWAL-KC-NHG-Nierfunctie-Scenario1-1"/>
+    <nts:includeDateT value="yes"/>
+    
+    <test id="lab-Scenarioset1">
+        <name value="Scenario 1"/>
+        <description value="Send a NHG coded panel of laboratory results along with a medication prescription."/>
+
+        <!--General asserts -->
+        <nts:include value="test.client.transaction" scope="common" sourceId="CERT-scenario-1-prescription-fixture"/>
+        <nts:include value="test.client.successfulTransaction" scope="common"/>
+        <nts:include value="assert.request.numResources" scope="common" resource="Specimen" count="1"/>
+        <nts:include value="assert.request.numResources" scope="common" resource="Organization" count="1"/>
+        <nts:include value="assert.request.numResources" scope="common" resource="MedicationRequest" count="1" warningOnly="true"/>
+        <nts:include value="assert.request.numResources" scope="common" resource="Patient" count="1"/>
+        
+         <!-- Asserts for checking the Patient resource to be compliant with the wiki page on
+        https://informatiestandaarden.nictiz.nl/wiki/Lab:1.0.0_Testgegevens_SturenResultaten-->
+        <nts:include value="assert.identifier" direction="request" resource="Patient" system="http://fhir.nl/fhir/NamingSystem/bsn" identifier="999910966"/> 
+        
+
+        <!--Initial asserts to check the lab result codes and status-->
+        <nts:include value="assert.numberofcodes" direction="request" resource="Observation" system="https://referentiemodel.nhg.org/tabellen/nhg-tabel-45-diagnostische-bepalingen" count="1"/>
```

Comment:
Waar kan ik dit terug vinden?

Reply:
Het gaat hier om het aantal keer dat er een Observation met een code van type NHG meegestuurd gaat worden in de Bundle. In het bericht is er in de hele Bundle, maar 1 Observation resource, vandaar dat hier count op 1 staat.